modules/rosa-policy-customer-responsibility.adoc

```diff
@@ -7,30 +7,50 @@
+|**Red Hat**
+
+- Provision clusters with OpenShift components installed so that customers can access the OpenShift and Kubernetes APIs to deploy and manage containerized applications.
 - Create clusters with image pull secrets so that customer deployments can pull images from the Red Hat Container Catalog registry.
 - Provide access to OpenShift APIs that a customer can use to set up Operators to add community, third-party, and Red Hat services to the cluster.
 - Provide storage classes and plugins to support persistent volumes for use with customer applications.
 - Provide a container image registry so customers can securely store application container images on the cluster to deploy and manage applications.
+
+**AWS**
+
+- Provide Amazon EBS to support persistent volumes for use with customer applications.
+
+- Provide Amazon S3 to support Red Hat provisioning of the container image registry.
+
 |- Maintain responsibility for customer and third-party applications, data, and their complete lifecycle.
 - If a customer adds Red Hat, community, third-party, their own, or other services to the cluster by using Operators or external images, the customer is responsible for these services and for working with the appropriate provider, including Red Hat, to troubleshoot any issues.
-- Use the provided tools and features to configure and deploy; keep up to date; set up resource requests and limits; size the cluster to have enough resources to run apps; set up permissions; integrate with other services; manage any image streams or templates that the customer deploys; externally serve; save, back up, and restore data; and otherwise manage their highly available and resilient workloads.
-- Maintain responsibility for monitoring the applications run on {product-title}, including installing and operating software to gather metrics and create alerts.
+- Use the provided tools and features to link:https://docs.openshift.com/container-platform/4.12/architecture/understanding-development.html#application-types[configure and deploy]; link:https://docs.openshift.com/container-platform/4.12/applications/deployments/deployment-strategies.html[keep up to date]; link:https://docs.openshift.com/dedicated/welcome/index.html[set up resource requests and limits]; link:https://docs.openshift.com/dedicated/welcome/index.html[size the cluster to have enough resources to run apps]; link:https://docs.openshift.com/dedicated/welcome/index.html[set up permissions]; integrate with other services; link:https://docs.openshift.com/container-platform/4.5/openshift_images/images-understand.html[manage any image streams or templates that the customer deploys]; link:https://docs.openshift.com/dedicated/welcome/index.html[externally serve]; save, back up, and restore data; and otherwise manage their highly available and resilient workloads.
```

Comment:
Some of these links direct to the OSD welcome page. Is this intentional?
- set up resource requests and limits
- size the cluster to have enough resources to run apps
- integrate with other services
- externally serve

Double check the rest of the links, too, please!

Reply:
Hello @eohartman,
   Thanks for the review and notes. Yes, those links were intentionally linking to the OSD welcome page per AWS feedback. That being said, I will double (triple?) check to see if that does indeed make the most sense. Thanks!